ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -156,19 +164,28 @@ const CompletedButton = ({buttonText, wasBypassed, style}: CompletedButtonInterf
     <ClrIcon shape='check' style={{marginRight: '0.3rem'}}/>{wasBypassed ? 'Bypassed' : buttonText}
   </Button>;
 
+const wasBypassed = (modules, moduleName) => switchCase(moduleName,
+    [AccessModule.DATAUSERCODEOFCONDUCT, () => !!getAccessModuleBypassTime(modules, AccessModule.DATAUSERCODEOFCONDUCT)],
+    [AccessModule.COMPLIANCETRAINING, () => !!getAccessModuleBypassTime(modules, AccessModule.COMPLIANCETRAINING)],
+    // these cannot be bypassed
+    [AccessModule.PROFILECONFIRMATION, () => false],
+    [AccessModule.PUBLICATIONCONFIRMATION, () => false]);
+
+
 interface ActionButtonInterface {
-  isModuleExpiring: boolean;
-  wasBypassed: boolean;
+  modules: AccessModuleStatus[];
```

Comment:
ActionButtons and RenewalCards can derive this information on their own from the modules